department-of-veterans-affairs/abd-vro
# MCP-2656 PDF File Fix

## What was the problem?

PDF Generator was unable to download some styling files for fontawesome and bootstrap. These don't need to be downloaded constantly so they will be added to the repo

```
Fontconfig error: No writable cache directories
Loading pages (1/6)
Warning: Failed to load https://use.fontawesome.com/releases/v5.15.4/css/all.css (ignore)
Error: Failed to load https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css, with network status code 6 and http status code 0 - SSL handshake failed
Error: Failed to load https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js, with network status code 6 and http status code 0 - SSL handshake failed
Counting pages (2/6)                                               
Loading TOC (3/6)                                                           
Error: Failed to load https://cdnjs.cloudflare.com/ajax/libs/bootstrap/5.0.2/css/bootstrap.min.css, with network status code 6 and http status code 0 - SSL handshake failed
Error: Failed to load https://cdnjs.cloudflare.com/ajax/libs/bootstrap/5.0.2/css/bootstrap.min.css, with network status code 6 and http status code 0 - SSL handshake failed
Resolving links (4/6)                                              
Loading headers and footers (5/6)                                           
Printing pages (6/6)
Done                                                                      
Exit with code 1 due to network error: SslHandshakeFailedError
```

Associated tickets or Slack threads:

- [MCP-2656](https://amida.atlassian.net/browse/MCP-2656)

## How to test this PR
- Generate a PDF


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
